server/proxy/bindings.go

```diff
@@ -164,15 +177,26 @@ func (p *Proxy) scanAppBindings(app *apps.App, bindings []*apps.Binding, locPref
 			}
 		}
 
+		// First level of Channel Header
+		if fql == apps.LocationChannelHeader.Make(b.Location) {
+			// Must have an icon on webapp to show the icon
+			if b.Icon == "" && userAgent == "webapp" {
```

Comment:
What do we gain by only throwing a log message when a webapp user requests the bindings? Does it make sense to always throw the error as come clients (webapp) will have issue with such binding?

Reply:
In this block we are doing two things: log the error, and continue, so this binding is filtered. We could separate the conditions, but we would have an even deeper if chain.

Reply:
I'm inclined to both filter and log even if the userAgent is not `webapp`. If a dev developers an app only using mobile, them will never notice that something is off.

Reply:
Good point. Please consider my point as irrelevant.